src/pages/patientView/timeline2/timeline-data-utils.ts

```diff
@@ -17,8 +17,23 @@ export function downloadZippedTracks(events: ClinicalEvent[]) {
     });
 }
 
+export function groupTimelineData(events: ClinicalEvent[]) {
+    const groupedData = _.groupBy(events, d => d.eventType);
+
+    return _.mapValues(groupedData, (data, eventType) => {
+        return getRows(data);
+    });
```

Comment:
Should we just do this instead?

```suggestion
    return _.mapValues(groupedData, data => getRows(data));
```